src/main/java/uk/gov/companieshouse/model/dto/dissolution/DissolutionGetResponse.java

```diff
@@ -1,7 +1,7 @@
 package uk.gov.companieshouse.model.dto.dissolution;
 
-
 import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.model.db.dissolution.DissolutionVerdict;
```

Comment:
db model in an api model. The key reason behind keeping data models out of controllers is to keep api layer and db layer completely separate. It's a key concept in the controller -> service -> db pattern.